src/lib/questions.ts

```diff
@@ -37,14 +38,15 @@ export default ({ cwd }: { cwd: string }): PromptObject[] => [
         title: 'Yarn',
         description: 'Use Yarn as the package manager',
         value: 'yarn',
+        disabled: !isYarnGlobal()
       },
       {
         title: 'NPM',
         description: 'Use NPM as the package manager',
         value: 'npm',
       },
     ],
-    initial: 0,
+    initial: isYarnGlobal() ? 0 : 1,
```

Comment:
I checked the code on this package, are you sure this does not give false negatives on WSL ?

Reply:
unfortunately, I'm not sure = (